.github/workflows/maven.yml

```diff
@@ -4,8 +4,6 @@
 name: Java CI with Maven
 
 on:
-  push:
-    branches: [develop, release-**]
```

Comment:
The e2e workflow also includes the SWC build and runs on push to `develop` and `release-*`. Stop running this action on push, so SWC build is only run once.